tests/smoke.rs

```diff
@@ -88,7 +106,6 @@ fn test(
             cache_compressed,
             rafs_mode.parse().unwrap(),
             "api.sock".into(),
-            // FIXME: Currently no digest validation is implemented for rafs v6.
```

Comment:
Why remove this comment?

Reply:
It was not deleted, just moved it to where it first appeared.